src/gui/optionsdlg.ui

```diff
@@ -1769,6 +1769,19 @@
                 </layout>
                </item>
                <item>
+                <widget class="QPushButton" name="banListButton">
+                 <property name="sizePolicy">
+                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
+                   <horstretch>0</horstretch>
+                   <verstretch>0</verstretch>
+                  </sizepolicy>
+                 </property>
+                 <property name="text">
+                  <string>Open the list of manually banned IP addresses</string>
```

Comment:
If I deabbreviate your version, it sounds like a "manually banned Internet Protocols". It looks bad.
I know that for a network programmer "IP" and "IP address" became kind of synonyms, but it is different things.
Of course, if your insist, I will change this name to whatever you want. But we already have done several iterations of renaming with __evsh__ and __thalieht__.